source/common/upstream/eds.cc

```diff
@@ -116,10 +121,10 @@ void EdsClusterImpl::onConfigUpdate(const ResourceVector& resources, const std::
   onPreInitComplete();
 }
 
-bool EdsClusterImpl::updateHostsPerLocality(const uint32_t priority, const HostVector& new_hosts,
-                                            LocalityWeightsMap& locality_weights_map,
-                                            LocalityWeightsMap& new_locality_weights_map,
-                                            PriorityStateManager& priority_state_manager) {
+bool EdsClusterImpl::updateHostsPerLocality(
+    const uint32_t priority, const HostVector& new_hosts, LocalityWeightsMap& locality_weights_map,
+    LocalityWeightsMap& new_locality_weights_map, PriorityStateManager& priority_state_manager,
+    std::unordered_map<std::string, HostSharedPtr>& new_all_hosts) {
```

Comment:
`new_all_hosts` is a bit confusing here, was this meant to be `updated_hosts`?

Reply:
yeah names went through a few iterations, will change